spyder/app/start.py

```diff
@@ -53,7 +53,7 @@
 from spyder.app.cli_options import get_options
 from spyder.config.base import (get_conf_path, reset_config_files,
                                 running_under_pytest, is_conda_based_app)
-from spyder.utils.conda import get_conda_root_prefix
+from spyder.utils.conda import get_conda_root_prefix, is_conda_env
```

Comment:
I moved `is_conda_env` to `spyder_kernels.utils.pythonenv`. So, please import it from there instead, in a `Third-party` section above this one.